src/nunit-gui/Views/SettingsPage.cs

```diff
@@ -86,13 +80,19 @@ public virtual void ApplySettings()
 
         #endregion
 
+        void LoadAndSetSettings()
```

Comment:
Why two methods? When do you want settings to be loaded without setting the flag?